parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
From my understanding "lib64" was the old way of doing things. I've tested this solution on amd64 and it worked. Haven't tried on ARM yet.